pkg/drivers/hyperkit/driver.go

```diff
@@ -268,13 +264,22 @@ func (d *Driver) Start() error {
 		}
 
 		d.IPAddress, err = GetIPAddressByMACAddress(mac)
-		if err != nil {
-			return &commonutil.RetriableError{Err: err}
+		return err
+	}
+
+	// Implement a retry loop without calling any minikube code
+	for i := 0; i < 30; i++ {
+		err = getIP()
+		if err == nil {
+			break
 		}
-		return nil
+		if err.Error() != fmt.Sprintf(IPErrorMessage, mac) {
```

Comment:
hey, do you think it's worth having a custom error and testing the type? comparing error strings seems ugly :/

Reply:
yeah I wasn't wild about text parsing either, and the custom error type is easy to add. done.